crates/sovereign-sdk/full-node/db/sov-db/src/ledger_db/mod.rs

```diff
@@ -625,25 +626,27 @@ impl BatchProverLedgerOps for LedgerDB {
     }
 
     #[instrument(level = "trace", skip(self), err)]
-    fn get_latest_job_ids(&self, count: usize) -> anyhow::Result<Vec<Uuid>> {
+    fn get_latest_jobs(&self, count: usize) -> anyhow::Result<Vec<(Uuid, JobStatus)>> {
         let mut read_opts = ReadOptions::default();
         // Do not fill the cache with garbage data just to read ids
         read_opts.fill_cache(false);
 
         let mut iter = self
             .db
-            .iter_with_direction::<CommitmentIndicesByJobId>(read_opts, ScanDirection::Backward)?;
+            .iter_with_direction::<JobIdOfCommitment>(read_opts, ScanDirection::Backward)?;
```

Comment:
This should be `CommitmentIndicesByJobId`. `JobIdOfCommitment` contains duplicate job ids because multiple commitment indices might have the same job id:
comm idx 2 => job id "abc"
comm idx 3 => job id "abc"

Job ids are UUID v7 so they are timestamp sorted, hence the last key in `CommitmentIndicesByJobId` is the last job that is created.